src/character.cpp

```diff
@@ -8383,6 +8391,53 @@ void Character::update_stamina( int turns )
     set_stamina( std::min( std::max( get_stamina(), 0 ), max_stam ) );
 }
 
+int Character::get_cardio() const
+{
+    return cardio;
+}
+
+void Character::set_cardio( int new_cardio )
+{
+    cardio = new_cardio;
+}
+
+void Character::update_cardio()
```

Comment:
i would have this be a function that calculates cardio on the fly, instead of updating a cached value.

Reply:
It isn't caching cardio; it's adjusting cardio.